codex_of_the_damned/templates/archetypes/experimental/creeping-sabotage.html

```diff
@@ -0,0 +1,107 @@
+    {% endtrans %}
+</div>
+{% endblock %}
+
+{% block archetype %}
+{% trans trimmed
+	beast_leatherface=card("Beast, The Leatherface of Detroit"),
+	ebanezer_roush=card("Ebanezer Roush"),
+	gustave_brunnelle=card("Gustaphe Brunnelle"),
+
+	aid_from_bats=card("Aid from Bats"),
+	bums_rush=card("Bum's Rush"),
+	carrion_crows=card("Carrion Crows"),
+	creeping_sabotage=card("Creeping Sabotage"),
+	deep_song=card("Deep Song"),
+	earthshock=card("Earthshock"),
+	improvised_tactics=card("Improvised Tactics"),
+	slam=card("Slam"),
+	the_warrens=card("The Warrens"),
+	thrown_sewer_lid=card("Thrown Sewer Lid"),
+
+	actions=link("/strategy/fundamentals", _("Actions"), _anchor="AB"),
+	combat=link("/strategy/combat", _("Combat")),
+	intercept=link("/strategy/fundamentals", _("Intercept"), _anchor="stealth-and-intercept"),
+	modules=link("/strategy/deck-building", _("Modules"), _anchor="modules-and-density"),
+	range_management=link("/strategy/combat", _("Range management"), _anchor="range-management"),
+	stealth=link("/strategy/fundamentals", _("Stealth"), _anchor="stealth-and-intercept"),
+
+	nananimalism=link("/archetypes/nananimalism", "Nananimalism"),
+	euro_brujah=link("/archetypes/euro-brujah", "Euro Brujah"),
+
+	img_clan_nosferatu='<img src="%(url)s" />'|format(url=url_for('static', filename='img/icons/clan-nosferatu.svg'))|safe
+%}
+<h2>Highlights</h2>
+<p>
+	This is a <em>rush</em> deck based on mid-cap {{ POT }} vampires.
+</p>
+<p>
+	Ordinary rush deck uses {{ ANI }} as the key discipline because {{ deep_song }}
+	is both the means to rush and the means to oust the prey. Other {{ combat }} decks
+	can enter combat using vampires' special action, like {{ beast_leatherface }} for
+	example.
+</p>
+<p>
+	There is no alternative to {{ deep_song }} for {{ POT }} decks and there are only
+	two Nosferatu {{ img_clan_nosferatu }} able to enter combat as a special action:
+	{{ ebanezer_roush }} and {{ gustave_brunnelle }}.
+</p>
+<p>
+	This means that the deck uses {{ bums_rush }} as a means to rush and
+	{{ creeping_sabotage }} as a means to oust the prey.
+</p>
+
+<h2>Tips & Tricks</h2>
+<p>
+	As {{ creeping_sabotage }} only deals damage to the prey, it is less likely that other
+	players will try to use actions to destroy it. {{ bums_rush }} can then be used to
+	reduce the number of {{ actions }} that the prey can use to attempt to destroy it.
+</p>
+<p>
+	If a prey or a predator has a good {{ range_management }} capability, {{ stealth }} can
+	be used to secure the optional maneuver from {{ bums_rush }}.
+</p>
+<p>
+	Using {{ the_warrens }} early in the game can dissuade some Methuselah to use actions
+	to attempt to destroy {{ creeping_sabotage }}.
+</p>
+<p>
+	There are some flex slots in the combat module that deals with ranged combat:
+	{{ thrown_sewer_lid }}, {{ slam }}, {{ earthshock }}, {{ improvised_tactics }}. Choosing
+	one over the others depends on what type of ranged fights you are expecting.
```

Comment:
Could be more precise - IMO Sewer Lid is great against CEL-guns as we can't expect to out maneuver them. Slam is very good against ANI. Improvised tactics is about cycling easily (can always be played at no cost, can even get an additional card with aus) if you don't expect much range. Earthshock is moot I think (pay 1 to do 1 more - there's no bruise advantage here), but maybe to replace the Roundhouses in case of heavy range management?